src/MOI_wrapper.jl

```diff
@@ -279,6 +294,22 @@ function _check_ret(model::Optimizer, ret::Cint)
     return
 end
 
+# If you add a new error code that, when returned by GRBoptimize,
+# should be treated as a TerminationStatus by MOI, to the global `Dict`
+# below, then the rest of the code should pick up on this seamlessly.
+const _ERROR_TO_STATUS = Dict{Cint, Tuple{MOI.TerminationStatusCode, String}}([
+    # TerminationStatus, RawStatusString
+    Cint(10001) => (MOI.MEMORY_LIMIT, "Available memory was exhausted."),
```

Comment:
Can you use instead `GRB_ERROR_OUT_OF_MEMORY => `: https://github.com/jump-dev/Gurobi.jl/blob/a995c081b7717b2f1a3e842d4b9ca51fa817d144/src/gen91/libgrb_common.jl#L11

Reply:
Done.